--- pynacl-1.4.0/setup.py.orig	2020-05-25 21:08:38.000000000 +0200
+++ pynacl-1.4.0/setup.py	2021-02-16 19:15:25.587854544 +0100
@@ -35,8 +35,7 @@
 
 
 requirements = ["six"]
-setup_requirements = ["setuptools",
-                      "wheel"]
+setup_requirements = ["setuptools"]
 test_requirements = ["pytest>=3.2.1,!=3.3.0",
                      "hypothesis>=3.27.0"]
 docs_requirements = ["sphinx>=1.6.5",
